packages/cactus-plugin-satp-hermes/src/main/typescript/plugin-satp-hermes-gateway.ts

```diff
@@ -266,32 +263,32 @@ export class SATPGateway implements IPluginWebService, ICactusPlugin {
       pluginOptions.keyPair = Secp256k1Keys.generateKeyPairsBuffer();
     }
 
-    const id = uuidv4();
     if (!pluginOptions.gid) {
       pluginOptions.gid = {
-        id: id,
+        id: uuidv4(),
         pubKey: bufArray2HexStr(pluginOptions.keyPair.publicKey),
-        name: id,
+        name: uuidv4(),
         version: [
           {
-            Core: "v02",
-            Architecture: "v02",
-            Crash: "v02",
+            Core: SATP_VERSION,
+            Architecture: SATP_VERSION,
+            Crash: SATP_VERSION,
           },
         ],
-        supportedDLTs: [SupportedChain.FABRIC, SupportedChain.BESU],
+        supportedDLTs: [],
         proofID: "mockProofID1",
         gatewayServerPort: DEFAULT_PORT_GATEWAY_SERVER,
         gatewayClientPort: DEFAULT_PORT_GATEWAY_CLIENT,
+        gatewayOpenAPIPort: DEFAULT_PORT_GATEWAY_API,
         address: "http://localhost",
       };
     } else {
       if (!pluginOptions.gid.id) {
-        pluginOptions.gid.id = id;
+        pluginOptions.gid.id = uuidv4();
       }
 
       if (!pluginOptions.gid.name) {
-        pluginOptions.gid.name = id;
+        pluginOptions.gid.name = uuidv4();
```

Comment:
The same here, if there is a case of the id and the name not existing they are created with diferent uuidv4, I think they should be correlated